src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -38,6 +38,10 @@ See xref:configuration/global-properties.adoc[Global Properties] for more inform
 
 - The `@MessagingGateway` and `GatewayEndpointSpec` provided by the Java DSL now expose the `errorOnTimeout` property of the internal `MethodInvocationGateway` extension of the `MessagingGatewaySupport`.
 See xref:gateway.adoc#gateway-no-response[ Gateway Behavior When No response Arrives] for more information.
+
+- A new method `setCreateIndex(boolean)` has been introduced in `org.springframework.integration.mongodb.store.AbstractConfigurableMongoDbMessageStore` to disable or enable (default) the auto index creation.
```

Comment:
This is a MongoDB support change, so it deserves its own `[[x6.2-mongodb]]` section in the end of this doc. Similar to `[[x6.2-jdbc]]` over there.

Also: we don't need to use the package for the class name in the docs.